### DISCUSSION TITLE
Does changing acceleration affect trackpad?

### DISCUSSION THREAD

**Discussion by jonochar** [linearmouse/linearmouse#126] [General]
Hey, love the app! 

I was wondering if the acceleration and sensitivity options were also applied to the trackpad. I can’t tell right now because I haven’t been using my MacBook for the past year. Thanks!

**Comment by lujjjh** [comment]
Hey, jonochar!

For the time being, changing acceleration [does not affect trackpad](https://github.com/linearmouse/linearmouse/issues/65#issuecomment-1030527237).

I'm still working on per-device settings, and this feature can be tracked in #65.
